en/peripherals/t265_vio.md

```diff
@@ -0,0 +1,31 @@
+# T265 VIO Camera Setup Guide
+
+The [Intel Realsense Tracking Camera T265](https://www.intelrealsense.com/tracking-camera-t265/) is a [VIO](https://dev.px4.io/master/en/advanced/computer_vision.html) (or V-SLAM) camera produced by Intel.
+It provides odometry information which can be used to augment or replace other positioning systems on PX4.
+
+## Setup Instructions
+
+These instructions explain how to set up VIO using ROS (the supported method):
+
+- On the companion computer, install and configure [MAVROS](https://dev.px4.io/master/en/ros/mavros_installation.html) and verify it is connected successfully to the flight controller
+- Get the [VIO bridge ROS node from Auterion](https://github.com/Auterion/VIO_bridge)
```

Comment:
How? Just install it on the same computer? Doe sit need a particular location?

Reply:
It's a ROS node, so it needs to be configured the same way as other ROS nodes. This (probably) means "clone into your catkin workspace source folder and build". I'll add this to the README on the repo.